README.md

```diff
@@ -23,10 +25,21 @@ Now any templates ending in .emblem will be compiled as Ember-Handlebars templat
 You can overwrite config as the followings:
 
 ``` ruby
-Ember::Emblem::Template.configure do |config|
-  config.precompile = true
-
-  # You can overwrite other config
+if defined?(Ember::Emblem::Template)
```

Comment:
This code most likely will end up in a rails initializer. Since not loading the full asset stack in production because they all are precompiled is a good thing I think, using it without the defined? check would crash the application in production since the gems will not be loaded.